docs/examples/network/complete-network-example.py

```diff
@@ -143,11 +143,10 @@ def runHotgym():
     fiveStep = results[5]["predictedValue"]
     fiveStepConfidence = results[5]["predictionConfidence"]
 
-    print("1-step: {:16} ({:4.4}%)\t"
-          "5-step: {:16} ({:4.4}%)".format(oneStep,
-                                           oneStepConfidence * 100,
-                                           fiveStep,
-                                           fiveStepConfidence * 100))
+    result = (oneStep, oneStepConfidence * 100,
+              fiveStep, fiveStepConfidence * 100)
+    print "1-step: {:16} ({:4.4}%)\t 5-step: {:16} ({:4.4}%)".format(*result)
+    yield result
```

Comment:
I want access the data generated by `runHotgym()` methods in all 3 `complete-example.py`  in my unit test - the thing testing wether the 3 frameworks output the same prediction results. I'm adding a `yield` at the end of `runHotgym()` to turn this method into a data generator (of HTM predictions) so that I can access prediction results in the tests. 

I could have accumulated the prediction results in a list and returned the list. Let me know if you have a preference of how to do this.